fuel/converters/caltech101.py

```diff
@@ -0,0 +1,196 @@
+import os
+import h5py
+
+import numpy
+import scipy.misc
+
+from six.moves import range
+
+from fuel.converters.base import fill_hdf5_file, MissingInputFiles
+
+CATEGORIES = (
```

Comment:
101 lines just for the `CATEGORIES` tuple is quite long, could we condense that?